habitat_baselines/models/rcm/vln_rcm_policy.py

```diff
@@ -250,6 +271,16 @@ def forward(self, observations, rnn_hidden_states, prev_actions, masks):
             [state, text_embedding, rgb_embedding, depth_embedding], dim=1
         )
 
+        progress_hat = torch.tanh(self.progress_monitor(x))
```

Comment:
This can all go under the if guard -- that way you don't need the progress sensor enabled at all during eval.